libraries/rib-base/src/main/java/com/badoo/ribs/clienthelper/childawareness/ChildAwareRegistry.kt

```diff
@@ -0,0 +1,69 @@
+package com.badoo.ribs.clienthelper.childawareness
+
+import androidx.lifecycle.Lifecycle
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.plugin.NodeAware
+import com.badoo.ribs.core.plugin.SubtreeChangeAware
+import kotlin.reflect.KClass
+
+/**
+ * This interface provides simplified access to the Nodes children
+ * when you need to create connection between them without any intermediate processing.
+ */
+interface ChildAwareRegistry : SubtreeChangeAware, NodeAware {
```

Comment:
`Registry` suffix doesn't make it sound like a plugin though. Why not call it `ChildAware`?